src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -1,41 +1,112 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import environment from 'platform/utilities/environment';
 
-export const VetTecContactInformation = ({ institution }) => (
-  <div className="additional-information row vads-u-margin-y--1">
-    <div className="usa-width-one-half medium-6 columns">
-      <div className="physical-address usa-width-one-whole">
-        <h3>Physical address</h3>
-        <div>
-          {institution.physicalAddress1 && (
-            <div>{institution.physicalAddress1}</div>
-          )}
-          {institution.physicalAddress2 && (
-            <div>{institution.physicalAddress2}</div>
-          )}
-          {institution.physicalAddress3 && (
-            <div>{institution.physicalAddress3}</div>
-          )}
+const SCOContactInformation = props => (
```

Comment:
If this is going to be outside of the `VetTecContactInformation` component it should be in a separate file

Additionally the first `<div className="usa-width-one-half medium-6 columns">` does not have protection against the first element missing

Also both `<div className="usa-width-one-half medium-6 columns">` appear similar enough that this can be a function that takes an element in and returns the div

```
const ScoContact = (scos, header) => {
  if (scos) {
    return (
      <div className="usa-width-one-half medium-6 columns">
        <div className="physical-address usa-width-one-whole">
          <h3>{header}</h3>
          <div>
            <div>
              {scos.firstName} {scos[0].lastName}
            </div>
            <div>{scos.title}</div>
            <div>
              <a href={`mailto:${scos.email}`}>{scos.email}</a>
            </div>
            <div>
              <a href={`tel:+1${`${scos.phoneAreaCode}-${scos.phoneNumber}`}`}>
                {scos.phoneAreaCode}
                {'-'}
                {scos.phoneNumber}
              </a>
            </div>
          </div>
        </div>
      </div>
    );
  }
  return null;
};
```

Reply:
Ok, no problem. I'm going to plan to pull it into a separate file and address the other concerns by genericizing it a bit.